src/modules/launcher/Wox.Plugin/PluginPair.cs

```diff
@@ -81,9 +81,11 @@ public void Update(PowerLauncherPluginSettings setting, IPublicAPI api)
             Metadata.ActionKeyword = setting.ActionKeyword;
             Metadata.IsGlobal = setting.IsGlobal;
 
-            if (Plugin is ISettingProvider)
+            (Plugin as ISettingProvider)?.UpdateSettings(setting);
+
+            if (IsPluginInitialized && !Metadata.Disabled)
             {
-                (Plugin as ISettingProvider).UpdateSettings(setting);
+                (Plugin as IReloadable)?.ReloadData();
```

Comment:
@cyberrex5 
Do you know this?
https://github.com/microsoft/PowerToys/blob/a96187bd042d893c0ce9f8cc2e3906a912bddbdb/src/modules/launcher/PowerLauncher/Plugin/PluginManager.cs#L121

Reply:
yes but this is not called anywhere